### ISSUE TITLE
Sidebar radius ?

### ISSUE THREAD

**Issue by notKamui** [elenapan/dotfiles#115]
Hi (again) !

My compositor works just fine, corners are rounded and transparent...that is, except for the sidebar.

I'm using your amarena sidebar. I there a reason that it doesn't work as intended ?

![2020 09 07-12 18 00 screenshot](https://user-images.githubusercontent.com/46132319/92378995-f9b97480-f106-11ea-9198-2e0248b341b9.png)

**Comment by elenapan** [comment]
The `amarena` sidebar reads the sidebar border radius from `beautiful.sidebar_border_radius`, in case you want to have different amounts of roundness in different themes. Which `theme` have you picked? (It is the very first choice in `rc.lua`). 

If you have picked `skyfall`, for example, you need to make this change in `~/.config/awesome/themes/skyfall/theme.lua`:

```diff
- theme.sidebar_border_radius = 0
+ theme.sidebar_border_radius = dpi(40)
```

The `amarena` theme has already set that border radius, so you can alternatively just switch to the `amarena` theme.

**Comment by notKamui** [comment]
I see, I chose the ephemeral theme. I was a bit confused as to where the value was retrieved from, because I indeed saw that value in the sidebar file, but could figure out where it was coming from.

It's actually my first time fiddling with AwesomeWM and Lua, but your code makes it generally really easy to read and understand :)

Sometimes I'm just dumb and get a bit lost.. ^^'

Thank you very much for your help!
